src/components/Board.vue

```diff
@@ -303,6 +300,24 @@
                 if (this.audioMap[sound]) {
                     this.audioMap[sound].play();
                 }
+            },
+            //Completely resets the game (score, scake position and food position)
+            resetGame(){
+                this.isGameOver = false;
+
+                this.score = 0;
+
+                this.speed = 500;
+
+                // Resets grid, snake position, and snake movement variables back to their defaults
+                this.squares = this.resetGrid();
+                this.direction = DIRECTION.RIGHT;
+                this.snakeHead = SNAKE.HEAD_START;
+                this.snakeBody = SNAKE.BODY_START;
+                this.canChangeDirection = true;
+                this.resetSnake();
+
+                this.squares[this.getFoodRandomCoords()] = SNAKE.FOOD;
```

Comment:
I like what you did here, but please undo all the other changes and remove the comments as they are no needed to understand the code 🙏